src/Tabs/Tab.js

```diff
@@ -0,0 +1,109 @@
+import React, { forwardRef } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import { colors } from '../theme.js'
+
+const noop = () => {}
+
+const Tab = forwardRef(
+    ({ icon, onClick, selected, disabled, fixed, children }, ref) => (
+        <button
+            className={`${cx({
+                selected,
+                disabled,
+                fixed,
+            })}`}
+            onClick={disabled ? noop : onClick}
```

Comment:
I think it's better to change `onClick`'s default value to undefined (by removing it from the defaultProps), that way no function will be executed. This line would change to `onClick={disabled ? undefined : onClick}`